scripts/check_enforcer/check_enforcer.py

```diff
@@ -209,8 +209,10 @@ def run_checks():
         merge_commit = (
             subprocess.check_output(["git", "log", "-1"]).decode("utf-8").split("\n")
         )
+        if snippet_debug == 1:
```

Comment:
when will this be 0?

Reply:
Just open this debug flag. If we don't want output and just get check enforcer output.
We can set this flag at the very top of the snippet

Reply:
But this is a good point, I'll make a switch here and let the yaml could control this snippet debug flag.